src/python/src/README.rst

```diff
@@ -0,0 +1,12 @@
+gRPC Python
+===========
+
+Package for GRPC Python.
+
+Dependencies
+------------
+
+Ensure that you've built the gRPC core from the root of the
+`gRPC git repo`_
```

Comment:
Can you update this so that this says to either install the released deb packages on linux or build from source on other platforms; cf these [instructiions](https://github.com/grpc/grpc-common/tree/master/node#install)